### ISSUE TITLE
False alarm of `Failed to do UI press on element: accessibility error kAXErrorAttributeUnsupported`

### ISSUE THREAD

**Issue by blindFS** [blindFS/Glyphlow#8]
Fools the app to trigger the fallback mouse clicking events, causing a troublesome double clicking on some toggles.
